extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -465,8 +465,8 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 		);
 
 		console.log(response);
```

Comment:
You should remove the (what I assume are) debug `console.log`s from checked in code so we don't have a bunch of that stuff cluttering up the console logs.